src/main/java/org/broadinstitute/barclay/help/DefaultDocWorkUnitHandler.java

```diff
@@ -667,8 +674,8 @@ private FieldDoc getFieldDoc(final ClassDoc classDoc, final String name, final b
      *         synonymous name.
      */
     private Pair<String, String> displayNames(String s1, String s2) {
-        s1 = s1 == null ? null : "-" + s1;
-        s2 = s2 == null ? null : "--" + s2;
+        s1 = ((s1 == null) || (s1.length() == 0)) ? null : "-" + s1;
```

Comment:
No prob - I noticed that some options were coming out as dashes when I was testing.